### PR TITLE
Changes simple animal death message

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
The way I did it doesn't allow for as much customization I guess but adding a new death message in New() for every mob seemed kind of silly

### CODE REVIEW

**Comment by Jordie0608** [comment]
But what if someone doesn't want their message to prepend with `[src]`

**Comment by coiax** [comment]
Byond's lack of delayed string formatting makes me sad.

**Comment by coiax** [comment]
Why can't we just do what python does with https://www.python.org/dev/peps/pep-3101/

**Comment by KorPhaeron** [comment]
@Jordie0608 

I guess I can close it then. The current code we have seems hacky and dumb though

**Comment by Jordie0608** [comment]
Tbh I'd make this change anyway and worry about my comment when someone actually wants to do that.

**Comment by KorPhaeron** [comment]
I gues I'll leave the choice of better code vs prettier messages to someone else then

File: code/modules/mob/living/simple_animal/hostile/skeleton.dm [hunk 1]
```diff
@@ -69,7 +69,7 @@
 	force_threshold = 10 //trying to simulate actually having armor
 	melee_damage_lower = 25
 	melee_damage_upper = 30
-	deathmessage = "The templar knight collaspes into a pile of bones, its gear clanging as it hits the ground!"
```
Comment (ChangelingRain) [inline code/modules/mob/living/simple_animal/hostile/skeleton.dm]: collapses

File: code/modules/mob/living/simple_animal/hostile/skeleton.dm [hunk 2]
```diff
@@ -46,7 +46,7 @@
 	gold_core_spawnable = 0
 	melee_damage_lower = 17
 	melee_damage_upper = 20
-	deathmessage = "The skeleton collaspes into a pile of bones, its gear falling to the floor!"
+	deathmessage = "collaspes into a pile of bones, its gear falling to the floor!"
```
Comment (ChangelingRain) [inline code/modules/mob/living/simple_animal/hostile/skeleton.dm]: same

File: code/modules/mob/living/simple_animal/hostile/skeleton.dm [hunk 3]
```diff
@@ -30,7 +30,7 @@
 	see_invisible = SEE_INVISIBLE_MINIMUM
 	see_in_dark = 8
 	layer = MOB_LAYER - 0.1
-	deathmessage = "The skeleton collaspes into a pile of bones!"
```
Comment (ChangelingRain) [inline code/modules/mob/living/simple_animal/hostile/skeleton.dm]: etc

File: code/modules/mob/living/simple_animal/hostile/zombie.dm [hunk 4]
```diff
@@ -153,6 +153,7 @@ mob/living/simple_animal/hostile/zombie/gymboss
 	spawn_text = "emerges from"
 	atmos_requirements = list("min_oxy" = 0, "max_oxy" = 0, "min_tox" = 0, "max_tox" = 0, "min_co2" = 0, "max_co2" = 0, "min_n2" = 0, "max_n2" = 0)
 	faction = list("zombie")
+	deathmessage = "collapes, stopping the flow of zombies!"
```
Comment (ChangelingRain) [inline code/modules/mob/living/simple_animal/hostile/zombie.dm]: etc

File: code/modules/mob/living/simple_animal/simple_animal.dm [hunk 5]
```diff
@@ -398,7 +398,7 @@
 		if(death_sound)
 			playsound(get_turf(src),death_sound, 200, 1)
 		if(deathmessage)
-			visible_message("<span class='danger'>[deathmessage]</span>")
+			visible_message("<span class='danger'>\the [src] [deathmessage]</span>")
```
Comment (ChangelingRain) [inline code/modules/mob/living/simple_animal/simple_animal.dm]: `\The [src]`

File: code/modules/mob/living/simple_animal/hostile/mining_mobs.dm [hunk 6]
```diff
@@ -875,19 +875,14 @@
...
 
 	animal_species = /mob/living/simple_animal/hostile/asteroid/gutlunch
 	childtype = list(/mob/living/simple_animal/hostile/asteroid/gutlunch/gubbuck = 45, /mob/living/simple_animal/hostile/asteroid/gutlunch/guthen = 55)
 
 	wanted_objects = list(/obj/effect/decal/cleanable/xenoblood/xgibs, /obj/effect/decal/cleanable/blood/gibs/)
 	var/obj/item/udder/gutlunch/udder = null
 
-
-/mob/living/simple_animal/hostile/asteroid/gutlunch/New()
-	udder = new()
```
Comment (coiax) [inline code/modules/mob/living/simple_animal/hostile/mining_mobs.dm]: Pretty sure the udder needs to be initialised.

**Comment by optimumtact** [comment]
Please fix the issues so I can marge this thanks :clap:

**Comment by KorPhaeron** [comment]
@optimumtact 

Should be okay now

**Pull request merged**